optuna_dashboard/_preferential_history.py

```diff
@@ -78,3 +79,23 @@ def report_history(
         key=key,
         value=json.dumps(history),
     )
+    return history_id
+
+
+def remove_history(study_id: int, storage: BaseStorage, uuid: str) -> None:
+    system_attrs = storage.get_study_system_attrs(study_id)
+    history: History = json.loads(system_attrs.get(_SYSTEM_ATTR_PREFIX_HISTORY + uuid, ""))
+    storage.set_study_system_attr(
+        study_id, _SYSTEM_ATTR_PREFIX_PREFERENCE + history["preference_id"], []
+    )
+
+
+def restore_history(study_id: int, storage: BaseStorage, uuid: str) -> None:
+    system_attrs = storage.get_study_system_attrs(study_id)
+    history: History = json.loads(system_attrs.get(_SYSTEM_ATTR_PREFIX_HISTORY + uuid, ""))
+    preferences = [
+        (best, history["clicked"]) for best in history["candidates"] if best != history["clicked"]
```

Comment:
```suggestion
        (better, history["clicked"]) for better in history["candidates"] if better != history["clicked"]
```
Nit: "better", not "best"